docs/HyperIndex/Advanced/reorgs-support.md

```diff
@@ -0,0 +1,39 @@
+---
+id: reorgs-support
+title: Reorgs Support
+sidebar_label: Reorgs Support
+slug: /reorgs-support
+---
+
+:::note
+This feature, while functional, is not optimized. If you choose to have this feature enabled, please be aware that historical indexing may slow down. We want to ensure blazing-fast indexing and are currently working on improving this. 
+:::
+
+Chain reorganizations are handled automatically by HyperIndex.
+
+To turn this feature on, you can set the ```rollback_on_reorg``` flag in your config.yaml to ```true```:
+```yaml
+rollback_on_reorg: true
```

Comment:
Its already on by default, you don't need to set it. 

You only need to set it to "off" is you don't want it

Reply:
kind adisagree with latest version it being off by default